aviary/subsystems/propulsion/engine_deck.py

```diff
@@ -466,51 +461,66 @@ def _check_data(self):
             # Check that units are the same. Variables have already been checked for valid
             # units, so it is assumed they are convertable. Prioritizes thrust units
             if engine_variables[RAM_DRAG] != engine_variables[GROSS_THRUST]:
-                self.data[RAM_DRAG] = convert_units(self.data,
-                                                    engine_variables[RAM_DRAG],
-                                                    engine_variables[GROSS_THRUST])
+                data[RAM_DRAG] = convert_units(
+                    original_data[RAM_DRAG],
+                    engine_variables[RAM_DRAG],
+                    engine_variables[GROSS_THRUST],
```

Comment:
should these be renamed to `engine_variable_units`?

Reply:
I don't think so - the point of the variable wasn't to store units specifically, but keep track of what variables are getting used by the engine. I used a dict as a convenient way to also track what units the data file asked for.

Basically questions like "does the engine use this variable" are cleanly communicated by `if XXX in engine_variables`, and would be less clear with a name change.